framework/moose.mk

```diff
@@ -395,6 +395,7 @@ install_harness:
 	@cp -f $(MOOSE_DIR)/scripts/moose_test_runner $(bin_install_dir)/moose_test_runner
 	@cp -f $(MOOSE_DIR)/framework/contrib/exodiff/exodiff $(moose_share_dir)/bin/
 	@cp -f $(MOOSE_DIR)/framework/include/base/MooseConfig.h $(moose_include_dir)/
+	@cp -f $(HIT_DIR)/hit.so $(python_install_dir)/
```

Comment:
This is now the only change to install. It just copies hit.so to the python directory, so you can "import hit"